src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceKind.kt

```diff
@@ -86,6 +87,16 @@ enum class LicenceKind(
   ) {
     override fun isHdc() = true
   },
+
+  TIME_SERVED(
+    TimeServedLicence::class.java,
+    { IN_PROGRESS },
+    { LicenceEventType.TIME_SERVED_CREATED },
+    { error("Time served licences cannot be copied") },
+    { LicenceEventType.TIME_SERVED_SUBMITTED },
```

Comment:
Don't think we need TIME_SERVED version of CREATED and SUBMITTED

Think it was a mistake to add them for HARD_STOP and one we didn't duplicate for HDC or PRRD.

Reply:
Will tidy these up in the follow on ticket around event types